conda_build/jinja_context.py

```diff
@@ -167,6 +167,43 @@ def load_npm():
         return json.load(pkg)
 
 
+def load_file_regex(config, load_file=None, regex_pattern=None, from_recipe_dir=False,
```

Comment:
Please make regex_pattern a non-default argument.  People really do need to specify it.